rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -441,7 +448,7 @@ if [ ${NB_COPIED_BINARIES} -gt 0 ];then echo "CFEngine binaries copied to workdi
 
 for i in cf-agent cf-promises cf-key cf-execd cf-serverd cf-monitord cf-runagent
 do
-  %{ruddervardir}/cfengine-community/bin/${i} -M | gzip > %{ruddervardir}/share/man/man8/${i}.8.gz
+  %{ruddervardir}/cfengine-community/bin/${i} -M | gzip > %{rudderdir}/share/man/man8/${i}.8.gz
```

Comment:
I don't see why we execute this command at postinst time instead of packaging time.
It would allow to keep the package clean with no binary alteration (think debsums).